src/App/App.js

```diff
@@ -28,19 +28,11 @@ class App extends Component {
     const formattedTitle = this.makeUpperCase(title);
     const movie = this.state.movies.find(movie => movie.title === formattedTitle);
     if (movie) {
-      this.setState({...this.state, selectedMovie: movie.id});
+      return movie.id;
```

Comment:
The path is handled in the search by name form now, so this just returns the id or false if there is no match